packages/esm-patient-registration-app/src/patient-registration/field/id/identifier-selection-overlay.tsx

```diff
@@ -21,42 +21,43 @@ const PatientIdentifierOverlay: React.FC<PatientIdentifierOverlayProps> = ({
   identifiers,
   remove,
 }) => {
-  const { patientIdentifiers } = useContext(ResourcesContext);
-  const [identifierTypes, setIdentifierTypes] = useState<PatientIdentifierType[]>([]);
+  const { identifierTypes } = useContext(ResourcesContext);
+  const [localIdentifierTypes, setLocalIdentifierTypes] = useState<Array<PatientIdentifierType>>([]);
   const [searchString, setSearchString] = useState<string>('');
   const { t } = useTranslation();
   const getIdentifierByTypeUuid = useCallback(
-    (identifierTypeUuid: string) => identifiers.find((identifier) => identifier.identifierType === identifierTypeUuid),
+    (identifierTypeUuid: string) =>
```

Comment:
I realize you didn't do this, but there's no need to use `useCallback` here. `useCallback` should be used when passing the function into another component.